Makefile

```diff
@@ -67,6 +67,16 @@ DEFS += -DNDEBUG
 DEFS += -DCONFIG_NFCT_PINS_AS_GPIOS
 DEFS += -DBUILD_VERSION='"$(BUILD_VERSION)"'
 DEFS += -DLFS2_NO_ASSERT
+DEFS += -DFIXED_POINT
+DEFS += -DUSE_KISS_FFT
+DEFS += -DEXPORT=
+DEFS += -DUSE_ALLOCA
+DEFS += -DOS_SUPPORT_CUSTOM
```

Comment:
It crashes if using malloc/calloc/realloc directly.
It needs MicroPython's m_malloc0 instead of calloc.